form.js

```diff
@@ -1,6 +1,7 @@
 const displayMsg = document.getElementById('form_message_error');
-const form = document.getElementById('contact-form');
+const form = document.getElementById('input');
 const email = document.getElementById('email');
+const formData = JSON.parse(localStorage.getItem('data'));
```

Comment:
- Please the popup window should be dynamic.
  - Please The JavaScript code should implement an array of objects to store JavaScript objects containing the information for each project.

   -  Please Each object should contain at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```